src/main/java/com/sap/cloudfoundry/client/facade/adapters/RawCloudEvent.java

```diff
@@ -13,34 +14,34 @@
 public abstract class RawCloudEvent extends RawCloudEntity<CloudEvent> {
 
     @Value.Parameter
-    public abstract Resource<EventEntity> getResource();
+    public abstract AuditEventResource getResource();
 
     @Override
     public CloudEvent derive() {
-        Resource<EventEntity> resource = getResource();
-        EventEntity entity = resource.getEntity();
+        AuditEventResource resource = getResource();
         return ImmutableCloudEvent.builder()
                                   .metadata(parseResourceMetadata(resource))
-                                  .actee(parseActee(entity))
-                                  .actor(parseActor(entity))
-                                  .timestamp(parseNullableDate(entity.getTimestamp()))
```

Comment:
What about the timestamp?

Reply:
In the v3 audit events api, the `created_at` metadata value of an audit event entity is the equivalent of the v2 event's `timestamp`